bzt/modules/blazemeter.py

```diff
@@ -226,10 +228,16 @@ def post_process(self):
         finally:
             self._postproc_phase2()
 
-        if self.client.results_url:
+        report_link = None
+        if self.client.token and self.public_report:
+            report_link = self.client.make_report_public()
```

Comment:
I think we need to show original link as we did before. If we make report public, let's print additional link. So user will see both. This might be very important for automation, since the only way to obtain masterID is to parse bzt.log